### DISCUSSION TITLE
My hello.html file/not uploaded

### DISCUSSION THREAD

**Discussion by hoojaoh** [community/community#22642] [New to GitHub]
<p>my one repositry , I added hello.html thru atom and pushed. but that hello.html file was not uploaded.</p>

**Comment by Wabri** [comment]
<p>Could you please give us some more information? like url of the repository!</p>

**Comment by hoojaoh** [comment]
<p>My  URL of the repository is <a href="https://github.com/hoojaoh/lecture0.git" rel="nofollow noopener">https://github.com/hoojaoh/lecture0.git</a></p>
<p><a href="https://github.com/hoojaoh" rel="nofollow noopener"><img src="https://avatars2.githubusercontent.com/u/12589064?s=60&amp;v=4" alt="@hoojaoh"></a><br>
<a href="https://github.com/hoojaoh/lecture0/commits?author=hoojaoh" rel="nofollow noopener">hoojaoh</a> <a href="https://github.com/hoojaoh/lecture0/commit/f1ab926e3257a983ff398b4a9e91de6122ea8e41" title="Added hello.html" rel="nofollow noopener">Added hello.html</a>         Latest commit <a href="https://github.com/hoojaoh/lecture0/commit/f1ab926e3257a983ff398b4a9e91de6122ea8e41" rel="nofollow noopener">f1ab926</a> 19 days ago</p>

**Comment by Wabri** [comment, marked as answer]
<p>You have to save the file, add the file to staged do the commit and then push. There is no way that the modified file were not updated.</p>
